content/soarian/dstu2/individuals/patient.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Patient | Soarian DSTU 2 API
+---
+ 
+# Patient
+
+* TOC
+{:toc}
+
+## Overview
+
+For the Patient resource query, the API returns a set of data stored for the patient as a result of Soarian admission, discharge, transfer (ADT) conversations.  The response represents the most current information about the patient charted in Soarian at the time of query.  
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_patient, :dstu2) %>
+
+## Custom Extensions
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+  Race Extension:
+
+ The URL for this custom extension is defined as  `http://hl7.org/fhir/StructureDefinition/us-core-race`
+
+ID                                | Value\[x] Type      | Description
+----------------------------------|---------------------|----------------------------------------------------------------------------------
+`Race`| [`CodeableConcept`]  | Note that CCDS specification indicates Race and Ehnicity are part of the CCDS but these are not part of the FHIR patient resource so I’ve added them as FHIR extensions.|
+ 
+ Example:
+ 
+<%= json(:SOARIAN_RACE_STATUS_EXTENSION) %>
+
+  Ehnicity Extension:
+
+ The URL for this custom extension is defined as  `http://hl7.org/fhir/StructureDefinition/us-core-ethnicity`
+
+ID                                | Value\[x] Type      | Description
+----------------------------------|---------------------|----------------------------------------------------------------------------------
+`Ethnicity`| [`CodeableConcept`]  | Note that CCDS specification indicates Race and Ehnicity are part of the CCDS but these are not part of the FHIR patient resource so I’ve added them as FHIR extensions.|
+ 
+  Example:
+ 
+<%= json(:SOARIAN_ETHNICITY_STATUS_EXTENSION) %>
+
+
+## Search      
+
+	GET /Patient?:parameters
+	
+_Implementation Notes_
+
+* The API returns the most current patient list.
+
+### Parameters
+
+No Patient resource specific search parameters are supported. To ensure an unambiguous patient-facing response, the most current information is always returned; date date/range parameters are not used for this resource.
```

Comment:
Remove this paragraph.

Also, I'm confused - it states no params are supported, but the example below shows a patientId param? If you can query by patientId, include that parameter in the table.